src/Mentat.Repository/Services/CardService.cs

```diff
@@ -109,6 +110,15 @@ public void SaveCard(string id, Card card)
             else
             {
                 card.Id = Guid.NewGuid().ToString();
+                if (card.IsPrivate == true)
```

Comment:
Why is privacy tied to a card being custom? If I am adding a card that I want to be public, why would I have to go through the extra step of making it private THEN toggling it to public?

Reply:
I forget exactly what is hidden or restricted when the isCustom attribute is set, but there is something it restricts. We did not want to remove isCustom because we did not want to break something, and we did not want to repurpose isCustom because then it would be tied to a function the name did not indicate. The solution was to tie isPrivate and isCustom together.

As I indicated in my other comment, some of this functionality was intended to work with card collections and making a card private was intended as a placeholder of sorts with the intention of preventing a card from being edited while in a collection but the code was not ready early enough to integrate with the card collections code.